pkg/frontend/admin_openshiftcluster_resources_list_test.go

```diff
@@ -115,43 +108,19 @@ func TestAdminListResourcesList(t *testing.T) {
 				subscriptions.EXPECT().Get(gomock.Any(), mockSubID).
 					Return(subscriptionDoc, nil)
 
-				resources.EXPECT().List(gomock.Any(), "resourceGroup eq 'test-cluster'", "", nil).Return([]mgmtfeatures.GenericResourceExpanded{
```

Comment:
these tests need to make it into adminactions/resourceslist_test.go

Reply:
I will work on this next.

Reply:
Added file `pkg/frontend/adminactions/resources_list_test.go`